app/timetables/models.py

```diff
@@ -1,3 +1,10 @@
 from __future__ import unicode_literals
 
 from django.db import models
+from app.shared.models import CommonInfo
```

Comment:
Add an extra line above here, we would want to visually separate imports from django and imports from the app (ref: two scoops of django)